packages/core/src/sass/app/_common/components/account-common.scss

```diff
@@ -0,0 +1,812 @@
+            }
+        }
+
+        .account-form__poi-confirm-example--status-message {
+            .dc-hint-box {
+                margin: 0;
+
+                .dc-icon {
+                    min-width: fit-content;
+                    align-self: baseline;
+                }
+            }
+        }
+    }
+
+    .min-height {
+        min-height: 50vh;
+
+        @include mobile {
+            min-height: unset;
+        }
+    }
+
+    &__container {
+        display: flex;
+        flex-direction: column;
+        justify-content: flex-start;
+        align-items: center;
+
+        @include mobile {
+            min-height: unset;
+            overflow-y: scroll;
+            justify-content: center;
+            padding-inline: 1.6rem;
+        }
+
+        &--reset {
+            align-items: normal;
+
+            @include mobile {
+                overflow-y: unset;
+                padding: 1.6rem;
+            }
+
+            .proof-of-identity__submit-button {
+                @include mobile {
+                    margin: unset;
+                    width: 100%;
+                }
+            }
+
+            .account-form__footer {
+                @include mobile {
+                    padding: 1.6rem 2.4rem;
+                }
+            }
+
+            .form-body {
+                @include desktop {
+                    z-index: 5;
+                }
+            }
+        }
+
+        .icon {
+            width: 128px;
+            height: 128px;
+            margin: 5.2rem 0 2.6rem;
+
+            @include mobile {
+                width: 72px;
+                height: 72px;
+                margin-top: 2.6rem;
+            }
+        }
+
+        .external-dropdown {
+            .dc-dropdown-list {
+                z-index: 5;
+            }
+        }
+    }
+
+    &__country-text {
+        margin-bottom: 1.6rem;
+    }
+
+    &__text {
+        @include mobile {
+            width: 94%;
+            text-align: center;
+            margin-bottom: 9rem;
+        }
+    }
+
+    &__dropdown-container {
+        margin-top: 1.6rem;
+    }
+
+    &__inner-container {
+        display: flex;
+        flex-direction: row;
+        justify-content: center;
+        align-items: center;
+        width: 100%;
+
+        @include mobile {
+            flex-direction: column;
+        }
+    }
+
+    &__header {
+        margin: 5.4rem 0 1.6rem;
+
+        @include mobile {
+            margin: 2.4rem 0 0.8rem;
+        }
+    }
+
+    &__footer.external-footer {
+        padding: unset;
+        position: unset;
+        top: unset;
+        bottom: unset;
+        border: none;
+        height: fit-content;
+        z-index: 4;
+    }
+
+    &__footer {
+        @include mobile {
+            display: flex;
+            flex-direction: row;
+            align-items: center;
+            justify-content: flex-end;
+            position: fixed;
+            bottom: 0;
+        }
+    }
+
+    &__footer-alert {
```

Comment:
@yauheni-deriv everything under with &__footer can be nested under `&__footer {`  Same for &__fieldset and &__sample-container down below :)